css/globals.css

```diff
@@ -0,0 +1,661 @@
+  width: fit-content;
+  border: 1px solid var(--border);
+  border-radius: 12px;
+  background-color: var(--bg);
+  padding: 0;
+  display: none;
+}
+
+.submenu-content li:first-child a {
+  border-radius: 12px 12px 0 0;
+}
+
+.submenu-content li:last-child a {
+  border-radius: 0px 0px 12px 12px;
+}
+
+.submenu.open .submenu-content {
+  display: initial;
+}
+
+.submenu-content li a {
+  display: block;
+  padding: 2px 20px 2px 20px;
+}
+
+#navbar .submenu-content a:hover {
+  background: var(--hover-bg);
+  text-decoration: none;
+}
+
+@media all and (max-width: 1110px) {
+  .submenu-content {
+    width: 100%;
+    position: static;
+    display: none;
+    margin-top: 7px;
+    background-color: var(--card-bg);
+    padding: 0;
+    margin: 0;
+    border: none;
+    border-radius: 0;
+    box-shadow: none;
+    max-height: 190px;
+    overflow-y: auto;
+    overflow-x: hidden;
+    transform: none;
+    cursor: pointer;
+  }
+
+  .submenu-content li>a {
+    width: 100%;
+  }
+
+  .submenu-content li:first-child a {
+    border-radius: 0;
+  }
+
+  .submenu-content li:last-child {
+    border-bottom: 0;
+  }
+
+  .submenu-content li:last-child a {
+    border-radius: 0;
+  }
+
+  .submenu-content.open {
+    display: inline-block;
+  }
+
+  .submenu-content li {
+    border-bottom: 1px solid var(--border);
+  }
+}
+
+
+/* -------- i18n Button --------*/
+
+#language-picker-menu {
+  display: none;
+}
+
+@media all and (max-width: 1110px) {
+  #language-picker-menu #navmenu>li:first-child {
+    display: flex;
+  }
+
+  #language-picker-menu #navmenu {
+    max-height: 70vh;
+    overflow-y: auto;
+    scrollbar-width: thin;
+    text-align: center;
+  }
+
+  #language-picker-menu {
+    display: block;
+    position: absolute;
+    top: 0;
+    right: 0;
+    width: 100%;
+    z-index: 1000;
+  }
+}
+
+/* -------- theme button --------*/
+
+#theme-icon-container {
+  cursor: pointer;
+}
+
+@media all and (max-width: 1110px) {
+  #theme-icon-container {
+    z-index: 10000;
+  }
+}
+
+/* -------- Print --------*/
+
+@media print {
+  header {
+    position: absolute;
+  }
+
+  #mobile-menu {
+    display: none;
+  }
+}
+
+/* ----------------- Blog --------------------- */
+
+#blog-side-menu-container {
+  position: fixed;
+  margin: 0;
+  padding: 0 10px 0 0;
+  top: 153px;
+  left: 30px;
+  height: 500px;
+  text-align: left;
+  font-size: 13px;
+  overflow-y: auto;
+}
+
+#blog-side-menu-container {
+  max-width: 210px;
+}
+
+#blog-side-menu-container li {
+  max-width: 100%;
+  margin-bottom: 5px;
+}
+
+#blog-side-menu-container>h3>a {
+  color: var(--fg)
+}
+
+#blog-side-menu {
+  list-style: none;
+  padding: 0;
+}
+
+#blog-side-menu>li>a {
+  color: var(--menu-grey);
+  font-weight: bold;
+  font-size: 13px;
+}
+
+@media all and (max-width: 1110px) {
+  #blog-side-menu-container {
+    display: none;
+  }
+}
+
+/* ----------------- Side Menu --------------------- */
+
+#menu {
+  position: fixed;
+  margin: 0;
+  padding: 0 10px 0 0;
```

Comment:
```suggestion
  padding-inline: 0.5rem;
  padding-block: 0.5rem;
```
old css blocks left side of li element on :focus-visible. And use logical css because we dont want to write separate css on translated page. These property follows directions and writing mode.

![image](https://github.com/user-attachments/assets/d7b34a54-baab-46c5-bfb5-e52a492d7dee)